projects/control-service/projects/helm_charts/pipelines-control-service/README.md

```diff
@@ -2,7 +2,8 @@
 Versatile Data Kit is a platform that enables Data Engineers to implement automated pull ingestion (E in ELT) and batch data transformation into a database (T in ELT).
 
 ## Prerequisites
-- Kubernetes 1.15+
+- Kubernetes 1.19<=x>1.25 works with no config changes
+- for kubernetes 1.25+ the datajob template needs to be changed in the [values.yaml](./values.yaml). Specifically `enabled` needs to be set to `true` and `apiVersion` needs to be set to `batch/v1`
```

Comment:
nit: maybe a quick note on k8s 1.25+ specifics linked (api changes k8s docs, for more info)